app/controllers/api/functions.php

```diff
@@ -1163,6 +1163,7 @@
         }
 
         $activate = (bool) filter_var($activate, FILTER_VALIDATE_BOOLEAN);
+        $type = $request->getHeader('x-sdk-language') === 'cli' ? 'cli' : 'manual';
```

Comment:
Any chance we can update a test case to check for this?

Reply:
Done

Reply:
The failing test is not relevant